setup.py

```diff
@@ -2003,16 +2003,9 @@ def detect_ctypes(self, inc_dirs, lib_dirs):
             ffi_inc = find_file('ffi.h', [], inc_dirs)
         if ffi_inc is not None:
```

Comment:
I haven't checked the implementation of `find_file`, but it looks like it probably takes care of the `os.path.exists` check and I'm not sure that the printed message is worth a whole lot in the grand spam of things.  Could you check on that and probably just remove this whole branch?